Runtime/Generated/ReactAction.cs

```diff
@@ -0,0 +1,605 @@
+// ------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+// ------------------------------------------------------------------------------>
+using System;
+using ReactUnity.Helpers;
+
+namespace ReactUnity.Helpers
+{
+  /// <summary>
+  ///   A wrapper class for the <see cref="ReactUnity.Helpers.Callback" /> class to provide convienient access to the
+  ///   to delegates and events in ReactUnity.
+  /// </summary>
+  class ReactAction
+  {
+      /// <summary>
+      ///   The underlying action.
+      /// </summary>
+      private Action _delegate { get; set; }
+
+      /// <summary>
+      ///   Adds the listener to the action, and returns a function that removes the listener. This method should be only called from React.
+      /// </summary>
+      [Obsolete("This method should be only called from React.", true)]
+      public Action AddListener(object callback)
+        {
+            var cb = Callback.From(callback);
+            Action listener = null;
+            listener = () =>
+            {
+                try
+                {
+                    cb.Call();
+                }
+                catch
+                {
+                    _delegate -= listener;
```

Comment:
It is not good to remove listener on `catch`. If this is about the refresh/hot-reload issue, there should be another way to fix this. For example, `Callback` can be marked as disposed when context is disposed. I will check what is viable.